geometry_msgs/Vector3 cam_pixel
---
geometry_msgs/PointStamped point